app/templates/_package.json

```diff
@@ -9,28 +9,28 @@
     "url": "<%= authorURL %>"
   },
   "scripts": {
+    "setup": "wp core download && npm install && bower install && npm run install-wp-plugins",
     "start": "grunt serve",
     "build": "grunt build",
-    "deploy": "git push && git push staging development",
+    "deploy": "git push origin development",
```

Comment:
`git push && git push staging development` will push to GitHub AND push to WPEngine (assuming that is the host). What's your thoughts on the above change?